evaluator/src/main/kotlin/LicenseView.kt

```diff
@@ -85,6 +85,19 @@ sealed class LicenseView {
         }
     }
 
+    /**
+     * Return only the concluded licenses if they exist, otherwise return detected licenses.
+     */
+    object ConcludedOrDetected : LicenseView() {
```

Comment:
There are tests for License views which should be adjusted as well, see LicenseViewTest.

Anyhow, the fact this introduces need for additional tests we may consider providing a more generic way to
create license views. If it was generic this change could look like the below which would not create any need for testing, imo the time is better spent in making it generic rather than adding more analog tests:

object ConcludedOrDetected : LicenseView( listOf(LicenseSource.CONCLUDED), listOf(LicenseSource.DETECTED  ) )

Reply:
@fviernau  I added tests to LicenseViewTest as requested